packages/x-components/src/components/icons/radiobutton-card-selected.vue

```diff
@@ -0,0 +1,21 @@
+<template functional>
+  <svg
+    width="32"
```

Comment:
In other icons, we don't set the width and the height, so remove them. Also, ask Cristina if she can export it with a viewbox of 0 0 16 16